.gitignore

```diff
@@ -0,0 +1,294 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
@estruyf is this repo only going to support these extensions? otherwise maybe we shouldn't have everything at the root

Reply:
This repo should be having the Office 365 CLI task as well.

Reply:
so we probably need to move all of that in a sub folder then?

Reply:
We can start with PowerShell only of course, but we need to make it possible to also support the CLI so that we can support other build agents.

Reply:
Why should this be an extra extension? Build tasks can be grouped within one extension. The grouping could be PnP.

For explanation: The fact, that this one task uses PowerShell has technically nothing to do with other tasks. They can live independently within the same published extension. The seperation and subfolder question come with the decision if you want to have literally two different extension:

- for PnP provisioning engine
- for Office 365 CLI

Reply:
thanks for the precision Elio.
So a couple of things then:
- it's really important that each task states all of it's demands to avoid scenarios like running on un-compatible platforms
- I don't have any issue with regrouping all the task under a single extension, I think it actually makes sense
- However naming of the extension, description and more have to make it clear that's not limited to PnP PS.
- And tasks must state clearly what they are leveraging
Does that make sense?

Reply:
Makes complete sense to me. I think I will create a new issue for that, so we can discuss the branding of the extension a bit further.

Because before publishing the following must be settled:

- metadata for extension
- metadata for PNP PowerShell task

Reply:
I opened an issue to discuss this here: #2